src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -947,10 +947,10 @@ trait Contexts { self: Analyzer =>
      *  if any such symbol is accessible from this context.
      */
     def importedAccessibleSymbol(imp: ImportInfo, name: Name): Symbol =
```

Comment:
I agree that the two code paths (`lookupImport` which is a real import versus `importedAccessibleSymbol` which is just a query) could have been better delineated in the original code, maybe by naming it `isThisSymbolAccessiblyImportableAskingForAFriend`, but the facade method was clearly intended to reduce clutter (of only one extra parameter). So the question is whether the facade method should be removed, the def at L953 renamed, and endowed with default args, for its one call site. Meh.

Since the query never looks for explicit import and never records, the params shouldn't be pushed to the callsite.

The facade method avoids defaults which would never be specified.